doc/cli/operator-sdk_run_ansible.md

```diff
@@ -15,6 +15,7 @@ operator-sdk run ansible [flags]
 ### Options
 
 ```
+      --ansible-roles-path string        Ansible path roles ($ANSIBLE_ROLES_PATH). By default it will be /roles in the current directory
```

Comment:
Did we want to add this in `run ansible`? I don't think users are supposed to use `run ansible` for local testing, just `up local`

Reply:
It is gen automatically. I will see how to fix.

Reply:
Solved good spot.

Reply:
By adding it to `run ansible`, it makes it possible for the Deployment to define an argument for this, and for developers it could be used via `operator-sdk up local --operator-flags="--ansible-roles-path=/my/path"`

Do we need a CLI argument at all? Or is the environment variable sufficient?